src/ocl/batchnormocl.cpp

```diff
@@ -37,6 +37,10 @@
 #include <miopen/convolution.hpp>
 #include <miopen/mlo_internal.hpp>
 #include <miopen/stringutils.hpp>
+#include <miopen/batchnorm/invoke_params.hpp>
```

Comment:
Are these changes tested for both FP32 and FP16?

Reply:
Yes.